EMF/common/on_actions/emf_on_actions.txt

```diff
@@ -109,7 +115,12 @@ on_adolescence = { # Checked on 12th birthday.
 on_post_birth_stillbirth = {
 	events = {
 		emf_birth.1 # Clear pregnancy state of mother after stillbirth
+<<<<<<< HEAD
 		emf_culture.17 # Clear emf_ai_never_convert_culture_unborn flag
+		emf_cadet.2 # Clear posthumous father-of-unborn cadet PETs now that we've finished pregnancy
+=======
+		emf_cadet.3 # Clear posthumous father-of-unborn cadet PETs now that we've finished pregnancy
+>>>>>>> fix for wrong scope in birth events / made same error :pdox: did
```

Comment:
There's something fishy going on here with those Git merge markers. Do you really want both emf_cadet.2 and emf_cadet.3 to trigger on birth?

Reply:
:laughing: 
Yeah, I fucked that one up big time. Pushing fix.